docs/constitution.md

```diff
@@ -36,24 +36,24 @@ This meeting will be carried out over a voice call, and all members will be noti
 
 ## Finances
 
-Finances for the **Hackathons for Schools Association** will be managed by the Core Team, who will be required to approve all expenditure on behalf of **HFSA**. All financial transactions involving **HFSA** must be recorded. This will primarily include travel to events where the member is representing **HFSA**. Only standard-class travel will be paid for, but a member can choose to upgrade if they are willing to pay the difference.
+Finances for the **Hackathons for Schools** will be managed by the Core Team, who will be required to approve all expenditure on behalf of **H4S**. All financial transactions involving **H4S** must be recorded. This will primarily include travel to events where the member is representing **H4S**. Only standard-class travel will be paid for, but a member can choose to upgrade if they are willing to pay the difference.
 
 ## Changes to the Constitution
 
 Changes can be made to the constitution by voting on it. A suggestion should be raised with the Core Team, who are then able to arrange a way for all members to voice their opinion and vote on the inclusion of the amendment.
 
-## Dissolution of the Hackathons for Schools Association
+## Dissolution of Hackathons for Schools
 
-In order for **HFSA** to be dissolved, the following procedure must be followed:
+In order for **H4S** to be dissolved, the following procedure must be followed:
 - Members notified of a Special General Meeting.
 - Members must vote on whether the dissolution will go ahead, a majority must be reached.
 - Finances must be in order:
     - Members must all be reimbursed for agreed expenditure.
-    - Remaining money should be given to a group with aims in line with those of the **HFSA**.
+    - Remaining money should be given to a group with aims in line with those of the **H4S**.
 
 ---
 
-This constitution has been agreed to by all members of the **Hackathons for Schools Association** and agreed by all members of the Core Team on the 23rd February 2020.
+This constitution has been agreed to by all members of the **Hackathons for Schools** and agreed by all members of the Core Team on the 23rd February 2020.
```

Comment:
Do you think this date should be updated given the changes?

Idk if its necessary or not